app/controllers/bundles_controller.rb

```diff
@@ -21,6 +24,11 @@ def redirect_if_disabled
 
   def index
     @bundles = Bundle.all
+    respond_with(@bundles)
```

Comment:
`Bundle` could also use some indexes. There are currently none, either in Cypress or in HDS. In particular, indexes on relations (ex: `has_many :product_tests, :dependent => :destroy` could get `index: true` added to it) would probably help, in addition to paring down the returned data.